src/BNC_parser/StrDataSample.py

```diff
@@ -0,0 +1,7 @@
+class StrDataSample:
+    def __init__(self, sentence = str(), language = 'ENG'):
+        self.sentence = sentence
+        self.language = language
+
+    sentence = str()
+    language = str()
```

Comment:
`sentence = str()`
`language = str() `
This two lines does not make any sense for me.